NeoForgeApi/build.gradle.kts

```diff
@@ -47,22 +47,14 @@ java {
 }
 
 dependencies {
-	implementation(
-		group = "net.neoforged",
-		name = "neoforge",
-		version = neoforgeVersion
-	)
 	dependencyProjects.forEach {
 		implementation(it)
 	}
 }
 
-minecraft {
-	accessTransformers {
-		// All minecraft configurations in the multi-project must be identical, including ATs,
-		// because of a ForgeGradle bug https://github.com/MinecraftForge/ForgeGradle/issues/844
-		file("../Forge/src/main/resources/META-INF/accesstransformer.cfg")
-	}
+neoForge {
+	version = neoforgeVersion
+	accessTransformers.from(file("../NeoForge/src/main/resources/META-INF/accesstransformer.cfg"))
```

Comment:
NeoForgeApi doesn't actually need ATs, this was just included as a workaround for https://github.com/MinecraftForge/ForgeGradle/issues/844

Is this still needed with MDG?

Reply:
It's not needed, no. I left it in to save one recompilation, but removing it is a bit cleaner. (Once it's in the cache it doesn't matter for subsequent runs.)

Reply:
We can leave it in, that sounds good. Can you add a comment above this line to explain why it's here?